app/services/media_storage/stored_path.rb

```diff
@@ -1,28 +1,45 @@
 # frozen_string_literal: true
 
 module MediaStorage
+  # StoredPath: module used by AzureAdapter for constructing the
+  # stored path/url of the media resource in azure
+  #
+  # Old S3 paths will include the panoptes-uploads domain prefix
+  # and the environment in their path, for example:
+  # panoptes-uploads.zooniverse.org/staging/subject_location/49f02f969a5.jpeg
+  # These URLs need to be rewritten such that domain prefix and env get removed
+  # Azure native paths do not need to be rewritten
   module StoredPath
-    # Construct the media access URL
-    # and ensure we remove the old aws s3 domain prefix in the stored paths
-    # but leave intact any non old s3 URL, i.e. new azure ones
+    # Construct path and join to the passed in URL
```

Comment:
URL here isn't really a good descriptor (or param name tbh). Really this is a domain prefix / hostname rather than a URL, https://developer.mozilla.org/en-US/docs/Web/API/URL

Perhaps we should rename here (and other places) while we're thinking about this?  

Perhaps update this as well as i don't think this is a URL either, perhaps a URL prefix? https://github.com/zooniverse/panoptes/blob/6287de9674f6c684c522e432a0016ff36ad27532/app/services/media_storage/azure_adapter.rb#L13